src/client/components/app-core/variables.css

```diff
@@ -1,3 +1,7 @@
 :root {
-  --action-color: red;
+  --action-color: #6a9ce4;
+  --neutral-color: #f5f5f5;
+  --dark-neutral: #4c4c4c;
+  --white: #fff;
```

Comment:
Maybe this is better?:
`--white` -> `--background-color`
`--black` -> `--text-color`